avocado/utils/pci.py

```diff
@@ -182,8 +182,9 @@ def get_slot_from_sysfs(full_pci_address):
     if not os.path.isfile("/proc/device-tree/%s/ibm,loc-code" % devspec):
         return
     slot = genio.read_file("/proc/device-tree/%s/ibm,loc-code" % devspec)
-    slot = re.match(r'((\w+)[.])+(\w+)-P(\d+)-C(\d+)|Slot(\d+)', slot).group()
-    return slot
+    slot = re.match(r'((\w+)[.])+(\w+)-P(\d+)-C(\d+)|Slot(\d+)|(\w+)[\s]Slot(\d+)', slot)
+    if slot:
+        return slot.group()
```

Comment:
Hello @abdhaleegit, there were several updates to this code, would you please add a selftest with couple of real-world examples? You can use `mock` to return the expected strings, feel free to inspire here: https://github.com/avocado-framework/avocado/pull/2634

Reply:
@abdhaleegit A better accomodating solution would be https://github.com/avocado-framework/avocado/pull/2380/commits/d86b977c7291cdec1ebb63bd58554a39d1971a93.

Can you include that case as well here please ?